test/k6/readme.md

```diff
@@ -33,11 +33,11 @@ Run test suite by specifying filename.
 
 For example:
 
->$> docker-compose run k6 run /src/tests/events/post.js -e tokenGeneratorUserName=*** -e tokenGeneratorUserPwd=*** -e env=***
+>$> podman compose run k6 run /src/tests/events/post.js -e tokenGeneratorUserName=*** -e tokenGeneratorUserPwd=*** -e env=***
```

Comment:
This command seems to require Powershell. Is that your exprience as well? I've been trying (git) bash without success. I have not tried regular command prompt.

Reply:
I use powershell as default, so that might be true. Have not tried cmd or bash